src/components/Filters/Scenarios.tsx

```diff
@@ -1,10 +1,48 @@
-import React, { Component } from 'react';
-import { Select } from 'antd';
-import { formatTitle } from '../../utils/utils';
-import { styles } from '../../utils/constants';
+import React, {Component} from 'react';
+import {Select} from 'antd';
+import {formatTitle} from '../../utils/utils';
+import {styles} from '../../utils/constants';
+import {Scenario} from "../../utils/constantsTypes";
```

Comment:
I think your linter got over-excited again - if you don't have a preference, for continuity, let's keep the `{ Component }` format with spacing!